### ISSUE TITLE
Training Loss convergance

### ISSUE THREAD

**Issue by Inferencer** [MRzzm/DINet#38]
I'm doing training but it's difficult to understand the loss convergance so I keep failing to get good results, how if you are training are you figuring it out?

**Comment by Saksham209** [comment]
HI @Inferencer , could you find an answer to this issue, I was trying to retrain DINet as well, but for the frame training stage (mouth region size = 64) the perception loss kind of saturates at around 3 without learning rate decay after 200 epochs, and I wanted to know about the values of perception loss we could aim for to say it has converged;.

**Comment by Inferencer** [comment]
I didn't no, I was going to add a live training chart to a collab but have given up untill I know what to look out for
